.travis.yml

```diff
@@ -16,6 +16,8 @@ stages:
   - name: unit-test
   # Builds are split into 'e2e-test' and 'build' to allow e2e tests to run first. If e2e fails, don't bother
   # building and pushing the images for the other architectures.
+  - name: minikube-e2e-test
+    if: (branch = main OR tag =~ ^v) AND fork = false AND type != cron
```

Comment:
@halim-lee Thanks for the PR. Missed it for RCO, but minikube tests, even when run against a PR, would use the _daily_ driver. So it wouldn't fail when a PR includes breaking changes. For now, let's skip running minikube tests for PRs. Run it for _main_ branch only and run it after _build-manifest_ stage (same conditions as the build-manifest stage). This way the _daily_ image would have been updated in Docker Hub and the minikube tests would use the new driver. If there is any issue, then the main branch build will fail. We can check the build result manually prior to creating the release driver. 

Please make the same changes for RCO. 

Let's do the same for OnePipeline, to begin with. After finishing more critical items for the 2Q release, we can revisit running the minikube tests for each PR.